sgx-types/src/ssa.rs

```diff
@@ -0,0 +1,135 @@
+#![allow(clippy::unreadable_literal)]
+
+#[derive(Copy, Clone, Debug)]
+pub enum Exception {
+    Divider,
```

Comment:
How do we know that these are all the types of `Exception`s? (Curious where this came from.)

Reply:
See above. I'd like to see if the `x86_64` crate has these types. If not, we should submit them and link against `x86_64`...